pwndbg/commands/context.py

```diff
@@ -352,9 +352,76 @@ def context(subcontext=None):
             out.flush()
 
 
+pwndbg.config.Parameter('show-compact-regs', False, 'whether to show a compact register view')
+pwndbg.config.Parameter('show-compact-regs-align', 20, 'the number of characters reserved for each register and value')
+pwndbg.config.Parameter('show-compact-regs-space', 4, 'the number of characters separating each register')
+
+
+def compact_regs(regs, width):
+    align = int(pwndbg.config.show_compact_regs_align)
+    space = int(pwndbg.config.show_compact_regs_space)
+    result = []
+
+    def calculate_padding(length):
+        return 0 if length % align == 0 else (align - (length % align))
+
+    def get_text_length(text):
+        '''Returns the text length while ignoring color-code escape sequences.'''
```

Comment:
Is this algorithm correct? Maybe we should use [`len(pwndbg.color.strip(text))`](https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/color/__init__.py) instead?

Reply:
Oh, wow. I didn't find this function. I will use `pwndb.color.strip` function as it already does the job.